wuzz.go

```diff
@@ -1016,7 +1055,19 @@ func main() {
 
 	initApp(app, g)
 
-	err = app.ParseArgs(g)
+	// load config (must be done *before* app.ParseArgs, as arguments
+	// should be able to override config values). An empty string passed
+	// to LoadConfig results in LoadConfig loading the default config
+	// location. If there is no config, the values in
+	// config.DefaultConfig will be used.
+	app.LoadConfig(configPath)
```

Comment:
I agree - I've been thinking about adding a one-line, read-only, non-selectable "debug" or "messages" view below where `search` is currently, to display various notifications to the user. A shortcut could bring up a popup that would show the full message history. It would work vaguely like the echo area and `*Messages*` buffer in emacs, the way I imagine it.